webui/tests/bin/createUserAndExperiments.py

```diff
@@ -43,6 +43,7 @@ def create_user(username, password=""):
         wait_for_process(p)
 
 
+print("setting up users..")
```

Comment:
nice, I like it

Reply:
and now the logs are in blue too. (.. for the main test script)

Reply:
love the ANSI color codes